docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -22,108 +22,90 @@ This is the signature of the constructor for the base Emitter class:
 def __init__(
         self,
         endpoint: str,
-        protocol: Literal["http", "https"] = "https",
+        protocol: HttpProtocol = "https",
```

Comment:
I think this was more useful in the previous form (`Literal["http", "https"]`) because now the reader doesn't know what the `HttpProtocol` type alias represents. The same for `Method`, `SuccessCallback`, and `FailureCallback`.